README.md

```diff
@@ -1 +1,62 @@
-# Todo_review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+
+> This project is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## Demo 
+Click here to see [Live](https://lionrouge1.github.io/To_do_list/)
+
+## Built With
+
+- HTML
```

Comment:
Add webpack after javascript here like this 

- html
- css
- javascript
- webpack